kong/reports.lua

```diff
@@ -448,6 +464,10 @@ return {
     if suffix then
       incr_counter(count_key .. ":" .. suffix)
     end
+
+    if ctx.route_match_cached then
```

Comment:
we can cache this var then use it, like `local route_match_cached = ctx.route_match_cached`